Makefile

```diff
@@ -1,3 +1,19 @@
+# or more contributor license agreements.  See the NOTICE file
```

Comment:
This line is missing:  # Licensed to the Apache Software Foundation (ASF) under one